tests/ctest/test_meta.c

```diff
@@ -300,12 +301,40 @@ static void checkTopArray (Key ** array, unsigned int size)
 	}
 	ksDel (done);
 }
+
+static void checkTopOrder1 (Key ** array)
+{
+	succeed_if (!strcmp (keyName (array[0]), "/d"), "Failed");
```

Comment:
defining your own macro for such cases is useful:
e.g. `succeed_if_top (0, "/d")`